backend/src/main/java/com/ody/auth/dto/request/AppleAuthRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.ody.auth.dto.request;
+
+import com.ody.mate.domain.Nickname;
+import com.ody.member.domain.DeviceToken;
+import com.ody.member.domain.Member;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+
+public record AppleAuthRequest(
+
+        @Schema(description = "디바이스 토큰", example = "devicetokendevicetoken")
+        @NotNull
+        String deviceToken,
+
+        @Schema(description = "회원 번호", example = "12345")
+        @NotNull
+        String providerId,
+
+        @Schema(description = "닉네임", example = "몽키건우")
+        @NotNull
+        String nickname,
+
+        @Schema(description = "프로필 사진 url", example = "imageurl")
```

Comment:
그럼 스키마 상에서 표기해주는 건 어떨까요?
```suggestion
        @Schema(description = "프로필 사진 url", example = "imageurl", nullable =true)
```

Reply:
기존 문서에서도 imageUrl이 필수 정보가 아니라는 점이 나타나서 별도로 표기를 추가하지는 않았습니다!

<img width="513" alt="image" src="https://github.com/user-attachments/assets/97335459-769b-48fd-a747-b65da56c4b4d" />